### PR TITLE
Fix tests for Windows, also "perl in space"

### PR DESCRIPTION
**Pull request by mohawk2**
Just as a by-the-by, a lot of this wouldn't be necessary if the scripts were all modulinos (#85) as testing would be much easier.

### CODE REVIEW

File: t/find.t [hunk 1]
```diff
@@ -104,17 +104,17 @@ sub check_find2perl {
 	my( $find2perl ) = @_;
 
 	ok( -e $find2perl, "find2perl exists at $find2perl" );
-	ok( -x $find2perl, "find2perl is executable $find2perl" );
+	ok( -x $find2perl, "find2perl is executable $find2perl" ) if $^O ne 'MSWin32';
```
Comment (briandfoy) [inline t/find.t:107]: Can you use SKIP blocks where instead?

**Comment by briandfoy** [comment]
Even if these modules were modulinos, I'd still want to do the interface tests just as you see here.

File: t/find.t (hunk 1 above)
Comment (mohawk2) [reply t/find.t]: Done. I also unified the nearly-identical `check_find{,2perl`.

**Comment by briandfoy** [comment]
I think this looks generally good, but give me a chance to test it on my side.

**Pull request merged**